.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -33,6 +33,7 @@ what is the error message you are seeing?
 
   - **CLI Version      :**
   - **Framework Version:**
+  - **Node.js Version:** <!-- Version of Node.js (run the command `node -v`) -->
```

Comment:
What about python/.net/java versions?

Reply:
good call, would save us some back and forth on a few occasions. tried to avoid adding another field, but not opinionated on formatting. let me know if you had something different in mind.